cirq/schedules/schedule_test.py

```diff
@@ -178,8 +178,6 @@ def test_query_overlapping_operations_exclusive():
     assert schedule.query(time=zero + 3.5 * ps, duration=ps) == []
 
 
-# In python 2, timedelta doesn't support multiplication with floats
```

Comment:
IIUC, this comment was added to justify an absence of a unit test that would not pass in python 2. Maybe implement it now to replace the comment you're removing?

Reply:
https://github.com/quantumlib/Cirq/issues/1618

Reply:
Actually, that comment was there to clarify the now missing annotation that that test should not run (and will not pass) in Python 2 because timedelta doesn't support multiplication with floats. The test itself uses that functionality and passed in Python 3. AFAIU, #1618 can be closed?